cogs/whisper.py

```diff
@@ -0,0 +1,27 @@
+import aiohttp
+import discord
+from discord.ext import commands
+
+class Whisper(commands.Cog):
+    """Whispers, for DCU Bot.
+    PM the bot with the !whisper prefix,
+    along with your message."""
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.session = aiohttp.ClientSession(loop=self.bot.loop)
```

Comment:
as pointed out by @flaree this isn't needed, and neither is the import associated with it.